documentation/automation.md

```diff
@@ -0,0 +1,142 @@
+# Cumulus Automation Inventory
+
+## Key Workflows
+
+This section summarizes the key workflows for developers, QEs, and technical
+writers.
+
+| Workflow                 | Flow                           | Org Type       | Managed | Namespaced |
+| ------------------------ | ------------------------------ | -------------- | ------- | ---------- |
+| Development              | `dev_org`                      | dev            |         |            |
+| Development (Namespaced) | `dev_org_namespaced`           | dev_namespaced |         | ✔          |
+| QA                       | `qa_org`                       | dev            |         |            |
+| QA (Namespaced)          | `qa_org_namespaced`            | dev            |         | ✔          |
+| Regression               | `regression_org`               | regression     | ✔       |            |
+| Trial (Clone of TSO)     | None (`cci org browser trial`) | trial          | ✔       |            |
+
+### Org Definitions
+
+NPSP includes many org definitions tailored for testing - in particular, beta testing - of the product in a variety of org shapes.
```

Comment:
Most of these orgs are identical in configuration. Do you think it would help to list out that configuration here, maybe just specifying where they differ?

Reply:
I think that's a great idea. Added.

Reply:
Thank you!